### PR TITLE
Feature/destop ui

### PR DESCRIPTION
**Pull request by Andileh**
Implementation of desktop UI.

### CODE REVIEW

**Review by ash-rabbit — APPROVED** [review]

File: waterlog/src/App.scss [hunk 1]
```diff
@@ -112,23 +112,66 @@ button {
...
+		display:block;
+	}
+	
+	.tabcontent{
+		width:80%;
+		margin:0px auto;
+	}
+
+	.map-icon-div-layer2{
+    left:80%;
```
Comment (FredJEhlers) [inline waterlog/src/App.scss:162]: Fix this entire file's indentation

File: waterlog/src/Pages/Home.js [hunk 2]
```diff
@@ -6,12 +6,14 @@ import TankLevels from "./../Components/TankLevels";
...
+        <div className="wastage-tank-wrapper">
+          <Wastage />
+          <TankLevels/>
+        </div>
       </div>
     );
   }
 }
-export default HomeComponent;
+export default HomeComponent;
```
Comment (FredJEhlers) [inline waterlog/src/Pages/Home.js]: empty line above this line and empty line EOF

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Pages/Home.js (hunk 2 above)
Comment (Andileh) [reply waterlog/src/Pages/Home.js]: done

File: waterlog/src/App.scss (hunk 1 above)
Comment (Andileh) [reply waterlog/src/App.scss:162]: that page is fine am using size 2 tab will show you the screen shot

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**